packages/glimmer-runtime/lib/dom/attribute-managers.ts

```diff
@@ -5,91 +5,140 @@ import {
   sanitizeAttributeValue,
   requiresSanitization
 } from './sanitized-values';
-import { normalizeProperty, normalizePropertyValue } from './props';
+import { normalizeProperty } from './props';
 import { SVG_NAMESPACE } from './helper';
 import { normalizeTextValue } from '../compiled/opcodes/content';
 import { Environment } from '../environment';
 
-export interface IChangeList {
+export interface AttributeManager {
   setAttribute(env: Environment, element: Simple.Element, attr: string, value: Opaque, namespace?: string): void;
   updateAttribute(env: Environment, element: Element, attr: string, value: Opaque, namespace?: string): void;
 }
 
-export function defaultChangeLists(element: Simple.Element, attr: string, isTrusting: boolean, namespace: string) {
+interface PropertyManager {
+  setAttribute(env: Environment, element: Simple.Element, attr: string, value: Opaque, namespace?: string): void;
+  updateAttribute(env: Environment, element: Element, attr: string, value: Opaque, namespace?: string): void;
+  removeProperty(env: Environment, element: Element, attr: string, value: Opaque, namespace?: string): void;
```

Comment:
The naming here is a tad odd.  Can we share interfaces with `AttributeManager` (I can't TS for 💩 )?  Naming the extra method `removeProperty` but all the others `*Attribute` "feels" weird.